js/pages/concept-sets/components/tabs/included-conceptsets.html

```diff
@@ -16,7 +16,10 @@
 <!-- ko if: $root.loadingIncluded -->
   <loading></loading>
 <!-- /ko -->
-<!-- ko ifnot: $root.loadingIncluded -->
+<!-- ko if: $component.hasErrors -->
+  <empty-state params="message: 'Could not load included concept sets'"></empty-state>
```

Comment:
With the `alert` approach, is there a need in the change? In which case are we going to reach the view?